src/web_interface/components/jinja_filter.py

```diff
@@ -125,7 +125,7 @@ def _setup_filters(self):
         self._app.jinja_env.filters['fix_cwe'] = fix_cwe
         self._app.jinja_env.filters['vulnerability_class'] = vulnerability_class
         self._app.jinja_env.filters['auth_enabled'] = self.check_auth
-        self._app.jinja_env.filters['md5_hash'] = lambda s: get_md5(s.encode())
+        self._app.jinja_env.filters['md5_hash'] = lambda s: get_md5(s)
```

Comment:
Do we need the lambda function any more?
`self._app.jinja_env.filters['md5_hash'] = get_md5` should do the same?